march_joint_inertia_controller/CMakeLists.txt

```diff
@@ -0,0 +1,108 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(march_joint_inertia_controller)
+
+# add_compile_options(-std=c++11)
+
+find_package(catkin REQUIRED COMPONENTS
+  controller_interface
+  pluginlib
+  roscpp
+  std_msgs
+  message_generation
+)
+
+# find_package(Boost REQUIRED COMPONENTS system)
```

Comment:
Again, please remove all commented out lines, since they might never be used. If they ever will be used someone can look up how to add messages.